### PR TITLE
Juniper

### PR DESCRIPTION
**Pull request by roaringsheep**

### CODE REVIEW

File: testfile.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+I wonder if the whole class can see this on Junipers branch now.
+
+Adding some extra info here.
+
```
Comment (roaringsheep) [inline testfile.txt:3]: Hey can you get rid of this new line here

File: testfile.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+Hello all.
+
+I'm making some changes here... maybe writing a login feature.
```
Comment (roaringsheep) [inline testfile.txt:6]: This line should go away

File: testfile.txt (hunk 1 above)
Comment (dgilliam2022) [reply testfile.txt:3]: Hey could we take out the extra lines here?

**Review by dgilliam2022 — CHANGES_REQUESTED** [review]
Could we take out extra lines?

File: testfile.txt (hunk 1 above)
Comment (roaringsheep) [reply testfile.txt:3]: Change made

**Review by dgilliam2022 — APPROVED** [review]

**Pull request merged**